amr-wind/incflo_advance.cpp

```diff
@@ -332,6 +333,9 @@ void incflo::ApplyPredictor (bool incremental_projection)
         icns().solve(dt_diff);
     }
 
+    if (m_sim.repo().field_exists("tke"))
+        amr_wind::field_ops::field_lower_bound(repo().get_field("tke"), 1.0e-15);
+        
```

Comment:
We need a better way to  handle this. This currently assumes that we always have TKE equation on.

Reply:
Done

Reply:
@gantech Let's not introduce such logic into `incflo_advance`, this should be handled by PDE. I'll work with you to get a proper design for this.